nbs/core.ipynb

```diff
@@ -64,9 +64,17 @@
     "def _build_fn_name(fn) -> str:\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    # from hierarchicalforecast.core import HierarchicalReconciliation
</pre><p>Why is this commented? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Thanks for catching this, I took the comment out.</p>